CMakeLists.txt

```diff
@@ -14,7 +14,6 @@ find_package(fastcdr REQUIRED)
 
 # Find Asio and Threads, both required for opening sockets from C++
 find_package(Asio REQUIRED)
-find_package(Threads REQUIRED)
```

Comment:
Why did you remove this line?
I found that on some linux the `pthread` library is not automatically linked.